Library/Homebrew/cmd/audit.rb

```diff
@@ -348,6 +348,7 @@ def audit_deps
         end
 
         dep.options.reject do |opt|
+          next true if opt.name == :devel
```

Comment:
I'll need to dig a bit deeper to figure out how to do this. Whenever I run `brew audit` on a sample formula that uses this, `spec.build.devel?` is always false unless I run `brew audit --devel`, which isn't what we want. Do you happen to know off-hand if there's somewhere else I should look to detect whether the dependency exists within a `devel` block?